src/main/java/org/jfrog/bamboo/release/action/condition/ReleasePromotionActionCondition.java

```diff
@@ -49,6 +52,38 @@ public boolean shouldDisplay(Map<String, Object> context) {
         return false;
     }
 
+    /**
+     * Extract plan of the master branch
+     *
+     * @param planKey The plan key in form of <PROJECT>-<BRANCH#>-<JOB>
+     * @return plan of the master branch
+     */
+    private Plan extractMasterPlanFromBranchPlan(PlanKey planKey) {
+        // Get chain key: <PROJECT>-<BRANCH#>
+        PlanKey chainKey = PlanKeys.getChainKeyIfJobKey(planKey);
+        if (chainKey == null) {
+            return null;
+        }
+
+        // Get plain of the branch plan
```

Comment:
I suggest removing this comment, because the code is actually clearer than the comment :)